backend/main.py

```
@@ -1745,6 +1952,316 @@ async def get_active_tasks():
     return models.ActiveTasksResponse(
         downloads=active_downloads,
         generations=active_generations,
+        finetunes=active_finetunes,
```

Ruff (0.15.2)

[error] Undefined name `active_finetunes`

(F821)